src/main/java/randoop/main/GenInputsAbstract.java

```diff
@@ -971,6 +985,18 @@ public void checkOptionsValid() {
               + " specified a class literal file and --use-class-literals=NONE");
     }
 
+    if (constant_mining && literals_level == ClassLiteralsMode.NONE) {
+      throw new RandoopUsageError(
+          "Invalid parameter combination:"
+              + " specified --constant-mining and --use-class-literals=NONE");
+    }
+
+    if (constant_mining && (constant_mining_probability < 0 || constant_mining_probability > 1)) {
+      throw new RandoopUsageError(
+          "Invalid parameter combination:"
+              + " specified --constant-mining and --constant-mining-probability is not in [0, 1]");
```

Comment:
I would think this should throw an exception anytime `constant_mining_probability` is out of range.  Or, should there be a check that `constant_mining_probability` is not set unless `--constant-mining` is passed?